src/hooks/use-projects.ts

```
@@ -0,0 +1,668 @@
+    }
+  }, [id, getProject]);
+
+  return {
+    project: currentProject,
+    getProject,
+    loading: loading.fetching,
+    error
+  };
+}
+
+export function useProjectValidation() {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useProjectValidation'. Consider to delete it or rename it.

'useProjectValidation' is defined here:


(lint/suspicious/noRedeclare)